basilisk/multiprocessing_worker.py

```
@@ -0,0 +1,115 @@
+	# Configure logging for the worker process
+	level = getattr(logging, log_level.upper(), logging.INFO)
+	logging.basicConfig(
+		level=level,
+		format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+	)
+
+
+def run_task_worker(
+	task: Callable, result_queue: Queue, cancel_flag, *args: Any, **kwargs: Any
+) -> None:
+	"""Run a task in a separate process.
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)